types/operator_metadata_test.go

```diff
@@ -15,8 +15,8 @@ func TestOperatorMetadata(t *testing.T) {
 		{
 			name: "Valid metadata",
 			metadata: OperatorMetadata{
-				Name:        "test",
-				Description: "test",
+				Name:        "Ethereum Utopia",
+				Description: "Madhur's first operator is best in this world",
```

Comment:
haha

Reply:
this was actually the description of the first operator I setup back in November :D